### DISCUSSION TITLE
关于页面移除的疑问

### DISCUSSION THREAD

**Discussion by Glonee** [note286/xduts#126] [Q&A] [answered]
使用的xduts版本:
```
Document Class: xduugthesis 2023/03/23 v6.1.4.2 Xidian University Undergraduate Thesis document class
```
使用以下代码移除封面:
```tex
\xdusetup{
    style = {
        remove-page = {封面}
    }
}
```
报错:
```
! LaTeX Error: The key 'xdu/style/remove-page' is unknown and is being
(LaTeX)        ignored.

For immediate help type H <return>.
 ...

l.27 }

? H

The module 'xdu/style' does not have a key called 'xdu/style/remove-page'.
Check that you have spelled the key name correctly.
```
是我的使用姿势不对吗？希望大佬解惑，感谢！

**Comment by note286** [comment, marked as answer]
本科暂不支持移除页面，文档里也没有引导本科可以移除。另外，本科有移除的需求吗？

![image](https://user-images.githubusercontent.com/16207292/236631794-cf3e599b-8890-44cc-a77d-020bcf7d9363.png)

**Reply by Glonee** [reply]
懂了，盲审需要空白封面，应该把xdusetup里设置的个人信息删了就行了吧

**Reply by note286** [reply]
是的。

![image](https://user-images.githubusercontent.com/16207292/236632095-8f7e7d2f-34ba-413c-b882-fd29b6dfc537.png)

**Reply by Glonee** [reply]
好的感谢大佬

**Reply by note286** [reply]
有啥建议和意见你可以新开讨论提，我都会考虑，今年有计划重新梳理一下，然后就不会再动了。

**Reply by Glonee** [reply]
OK，我是直接看到了文档里的5.21.2这一章，然后看到了可以页面移除，就以为页面移除这个功能是在本科和研究生论文里都可用的，感谢大佬及时解惑，如果可以在文档里标注一下就更好了。最后感谢大佬提供如此优秀的模板！

**Reply by note286** [reply]
我后续会优化一下文档结构。主要是不同文档类有很多公共的部分，我就都罗列出来了，在文档开头已经指路了，该文档类有哪些功能，后面就没再啰嗦。按道理应该是从头看的。

**Comment by PillarsZhang** [comment]
Acrobat和福昕是整治PDF的最强后盾...
其实本科的匿名功能也不支持，好在这个可以自己写命令
感谢大佬的模板呀~

**Reply by note286** [reply]
匿名是为了研究生隐藏个人成果中的个人信息，本科生是没有个人成果的。如果是封面等信息，使用注释的方式会更加方便。

**Reply by note286** [reply]
可以参考 https://github.com/note286/xduts/issues/61 ，之前也是我没考虑到，后续我会增加功能，近期没什么精力和时间了，2023届是赶不上了。本科的部分是2022届毕业的时候做的，2022届的一些内测同学也没提到这些，我也没想到这些，后续都会支持的。

**Reply by fuxiong0334** [reply]
谢谢你提到本科的删除页面和匿名功能不支持。删除页面封面可以自行删去，但是盲审要删去致谢的话，注释掉致谢还是会有致谢的题头，只能手动将内容替换或者删掉页面手动改页码。
